### ISSUE TITLE
Unwanted deletion of the note

### ISSUE THREAD

**Issue by pmslava** [laurent22/joplin#9148]
## Story

I had a note, created on Joplin Android. On Joplin Desktop, I decided I do not need this anymore and deleted it.
After that day, I forgot that I deleted this because it was still on Android. I worked all the day and put results into this note on Android. The next day on Android, synchronization started and deleted my local note without sending it to the syncing target.

## Environment

Joplin version: Android (2.12.3), Linux (2.12.19)
Platform: Android, Linux, Joplin Cloud
OS specifics: Android 14, Linux EndeavourOS

## Steps to reproduce

1. Create note on Android. 
2. Sync with target on Android. 
3. Sync with target on Desktop. 
4. Delete note on Desktop.
5. Work with note on Android. 
6. Sync with target on Android. 
7. Note with all work deleted without syncing.

## Describe what you expected to happen

If there is the deletion of the note on the sync target, but on the local device we have this note with more recent changes - do not delete the note on the local device and send it to the sync target.

An additional question is why it wasn't synchronization for the whole day of working with notes on an Android device.

## Logfile

[syncReport-1698392418955.txt](https://github.com/laurent22/joplin/files/13186312/syncReport-1698392418955.txt)

## P.S.

If there is a chance to restore my work from Android in some way - please tell me. The job I have done is not repeatable.
`27T09:16:02,30,"Synchronizer: ""Sync: deleteLocal: remote has been deleted: Note: (Local 532c4befe2e64174b63a39927e4737bb): (Remote 532c4befe2e64174b63a39927e4737bb.md)""`

**Comment by laurent22** [comment]
The note should be in the Conflict folder

**Comment by pmslava** [comment]
> The note should be in the Conflict folder

No, it wasn't put into the Conflict folder. As you can see by sync report in "P.S." the note was just deleted.

**Comment by laurent22** [comment]
You may be able to restore some of the content using this technique, but it's not sure: https://joplinapp.org/faq/#i-deleted-some-notes-by-accident-and-don-t-have-a-backup

**Comment by pmslava** [comment]
> but it's not sure

Yes, I saw this thread. But do we have a command pallet on Android Joplin? As a side note, I do not have root on the phone to reach DB itself.

Actually, I opened the issue not for the purpose of restoring this note (it would be good if we can, of course), but for preventing this from happening further. As I understood, there is no check on timestamps of last changes when a note is deleted locally by sync target.
